docs/data/migration/migration-pickers-v7/migration-pickers-v7.md

```diff
@@ -256,3 +256,32 @@ const theme = createTheme({
   },
 });
 ```
+
+## Stop passing `utils` and the date/time object to some translation keys
+
+The following translation keys expect to receive a function: `openDatePickerDialogue`, `openTimePickerDialogue`, or `clockLabelText`. If you are customizing the translation of those keys, you no longer need to provide `utils` and the date/time object to it, but only the formatted date/time value as a string. This affects the customization of those translation keys that on different contexts:
+
+Translations on a single component:
+
+```diff
+-<DatePicker localeText={{ openDatePickerDialogue: (date, utils) => string; }} />
++<DatePicker localeText={{ openDatePickerDialogue: (formattedDate) => string; }} />
+```
+
+Translations on custom components passed as slots:
+
+```diff
+ const translations = useTranslations();
+-const openDatePickerDialogue = translations.openTimePickerDialogue(value, {} as any, value == null ? null : value.format('MM/DD/YYY'));
++const openDatePickerDialogue = translations.openDatePickerDialogue(value == null ? null : value.format('MM/DD/YYY'));
+```
+
+Notice that `clockLabelText` expects an additional parameter with the time view.
```

Comment:
I would suggest just doing on diff per translation key.
The migration guide can be very lengthy if it helps people migrate (see [this section](https://mui.com/x/migration/migration-pickers-v6/#adapters-internal-changes) for a very long migration content :laughing: )


So something like:


````
The following translation key no longer receive the `utils` and the date object:

- `openDatePickerDialogue`:

  ```diff
  // If you are setting a custom value for this translation key in a picker component or in the `LocalizationProvider`
  -<DatePicker localeText={{ openDatePickerDialogue: (date, utils) => string; }} />
  +<DatePicker localeText={{ openDatePickerDialogue: (formattedDate) => string; }} />

  // If you using this translation key in a custom component
   const translations = useTranslations();
  -const openDatePickerDialogue = translations.openTimePickerDialogue(value, {} as any, value == null ? null : value.format('MM/DD/YYY'));
  +const openDatePickerDialogue = translations.openDatePickerDialogue(value == null ? null : value.format('MM/DD/YYY'));
  ```

  - `openTimePickerDialogue`:

  ```diff
  ```
````